hrm-domain/hrm-service/migrations/20240105181735-profile-flag-valid-until.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (C) 2021-2023 Technology Matters
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    /// ProfileSections
+    await queryInterface.sequelize.query(`
+      ALTER TABLE public."ProfilesToProfileFlags" ADD COLUMN IF NOT EXISTS "validUntil" timestamp with time zone; 
```

Comment:
Sometimes it can be more advantageous to record the start date of a validity period, and calculate the end time based on configuration when you evaluate it. It makes applying changes to the validity period more straightforward because you only need to update the config number rather than having to update the end times of all the existing records, but this approach might suit this specific use case better, just a suggestion